### DISCUSSION TITLE
Upgrading to 25.0.0 or 25.0.1 throws "java.lang.NullPointerException: Cannot invoke "org.keycloak.models.RealmModel.getMasterAdminClient()" because "realm" is null

### DISCUSSION THREAD

**Discussion by Snehalssg** [keycloak/keycloak#30974] [Q&A]
We want to upgrade to 25.0.0 version to reduce security vulnerabilities. 

We have downloaded 25.0.0 quarkus distribution and trying to test it with all the clients using the same setup as we have it for older version of keycloak (19.0.3).

Once we login, keycloak admin console throws below error only for master relam. 
![image](https://github.com/keycloak/keycloak/assets/46847027/1dc023c4-baf9-49f4-9621-e5438536febe)

As per logs, it throws null pointer exception

**2024-07-01 18:49:23,893 ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-183) Uncaught server error: java.lang.NullPointerException: Cannot invoke "org.keycloak.models.RealmModel.getMasterAdminClient()" because "realm" is null
	at org.keycloak.services.resources.admin.AdminConsole.addMasterRealmAccess(AdminConsole.java:281)
	at org.keycloak.services.resources.admin.AdminConsole.whoAmI(AdminConsole.java:253)
	at org.keycloak.services.resources.admin.AdminConsole$quarkusrestinvoker$whoAmI_eaeea38b34c9e4843e24a94ba86a1ba4c8cf0402.invoke(Unknown Source)**
	
The same setup is working fine for keycloak version 19.0.3 and 24.0.3 version.

Noticed same error when tried to use 25.0.1.

**Comment by jonkoops** [comment]
Did you migrate straight from 19 to 25? If so, please try migrating step by step using the next major.
